### PR TITLE
Adding big endian support for gosigar

### PR DESCRIPTION
**Pull request by Nayana-ibm**
Hi ,
This is with reference https://github.com/elastic/gosigar/issues/73#issuecomment-306795092
Added support for big endian. Please review.

### CODE REVIEW

**Comment by Nayana-ibm** [comment]
I have already signed CLA with my github id.

File: endian/endian.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package endian
```
Comment (andrewkroh) [inline endian/endian.go]: Can you move this over to the `sys` package?

File: endian/endian.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import "encoding/binary"
+import "unsafe"
```
Comment (andrewkroh) [inline endian/endian.go]: Please put these into a single import block.

```
import (
	"encoding/binary"
	"unsafe"
)
```

File: endian/endian.go (hunk 1 above)
Comment (andrewkroh) [reply endian/endian.go]: To `sys/endian.go` and usage will be `sys.GetEndian()`.

Comment (Nayana-ibm) [reply endian/endian.go]: done. incorporated changes.

**Comment by ruflin** [comment]
@Nayana-ibm It seems you are pushing commits with 2 different users which causes issues with the CLA checker.

**Comment by ruflin** [comment]
@Nayana-ibm I just had a look at our CLA checker and strangely it seems you are in the list and it contains your name and email, but no Github handle? Any chance you could try to sign the CLA again and make sure you push commits always from the same account?

**Comment by Nayana-ibm** [comment]
@ruflin  I have again signed CLA with GitHub as Nayana-ibm

**Comment by karmi** [comment]
Hi @Nayana-ibm, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/gosigar/pull/98.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by ruflin** [comment]
@Nayana-ibm I had to do some manual changes to the CLA check as it seems to be the company CLA. It's now mostly working. The part that the CLA checker still throws up is that in this PR there seem to be 2 different emails. See also Github shows for some commits different icons. Any chance you could rebase and set the same commiter/email for all commits? Could it be that you push code from 2 different machines?

screen shot 2018-03-02 at 09 47 31

File: endian/endian.go (hunk 1 above)
Comment (andrewkroh) [reply endian/endian.go]: Thanks for renaming. Can you please remove the `endian/endian.go` file too.

**Comment by Nayana-ibm** [comment]
@ruflin @andrewkroh  I am working on creating a separate PR with correct email id .

**Comment by Nayana-ibm** [comment]
Re-created PR : https://github.com/elastic/gosigar/pull/100

**Comment by Nayana-ibm** [comment]
closing this

**Pull request closed without merging**